sdk/metric/aggregation/aggregation.go

```diff
@@ -164,3 +164,65 @@ func (h ExplicitBucketHistogram) Copy() Aggregation {
 		NoMinMax:   h.NoMinMax,
 	}
 }
+
+// ExponentialHistogram is an aggregation that summarizes a set of
+// measurements as an histogram with buckets widths that grow exponentially.
+type ExponentialHistogram struct {
```

Comment:
The spec doesn't have normative language about the name of the type.  It refers to them both as `Base2 Exponential Bucket Histogram` (in the sdk doc), and Exponential Histogram (in the data-model doc).

Languages don't seem to agree, Java takes the `Base2ExponentialHistogramAggregator` but python the simpler `ExponentialBucketHistogramAggregation`.

I would prefer simpler titles, but if you think there is a better name put it as a suggestion, and I will update the rest of the code to match.

Reply:
If there is a possibility that a different base exponential might be added in the future having this named something like

```suggestion
type Base2ExponentialHistogram struct {
```

will help make a distinction between this and the future one.

If there is isn't any plans to add an additional exponential histogram the future then the way it is makes sense.

Reply:
I don't think there ever will be a different base.

If there is could the following ones be named with the specific base?

Reply:
Resolving. If we add a future expo hist, that one will have the prefix.

Reply:
I'm re-opening this because I see that the specification made a deliberate choice to rename with the prefix of `Base2`: https://github.com/open-telemetry/opentelemetry-specification/pull/3041/files#diff-32d5f289beed96900fa1e6e69a3ee9cc0b6503ce8883672011c44bc772c0fa2aL26-R26

I think we should follow and make the rename to follow [Java](https://github.com/open-telemetry/opentelemetry-java/blob/ad6cbfc4b49b1c996b588a3942d665034468da21/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/Base2ExponentialHistogramAggregation.java#L29), [.NET](https://github.com/open-telemetry/opentelemetry-dotnet/blob/02a40fc8b3d83a01ae6a7f007f0cea885e781c98/src/OpenTelemetry/Metrics/Base2ExponentialBucketHistogram.cs#L28), and [C++](https://github.com/open-telemetry/opentelemetry-cpp/blob/0c5f90dea5147461df4ff7d32cbb5d42e5011daa/sdk/include/opentelemetry/sdk/metrics/aggregation/base2_exponential_histogram_indexer.h#L20).